examples/overview.php

```diff
@@ -115,16 +115,17 @@ public function ensureUser() { $this->user->ensure(); }
 $user->setCreatedDate( new DateTime( '2013-10-10' ) );
 $userManager = new UserManager();
 
-for ( $i = 1; $i < 6; $i++ ) {
+for ( $i = 1; $i < 12; $i++ ) {
+	$number = str_pad($input, 2, "0", STR_PAD_LEFT);
```

Comment:
That's actually a misplaced auto completion. That's what happens if I hurry to commit :D